templates/search_results.hbs

```diff
@@ -10,12 +10,13 @@
       {{#if help_center_filters}}
         <section class="filters-in-section collapsible-sidebar">
           <h3 class="collapsible-sidebar-title sidenav-title">{{t 'filter_by_help_center'}}</h3>
-          <ul>
+          <ul class="multibrand-filter-list multibrand-filter-list--collapsed">
             {{#each help_center_filters}}
               <li>
                 <a href="{{url}}" class="sidenav-item" aria-selected="{{selected}}">{{name}} ({{count}})</a>
               </li>
             {{/each}}
+            <a tabindex="0" class="see-all-filters-trigger" aria-hidden="true" title="{{t 'see_more'}}">{{t 'see_more'}}</a>
```

Comment:
Please use `button` instead for `a`.

And `tabindex="0"` shouldn't be necessary.

Reply:
@VilhelmNielsen thanks for reviewing this. according to the design it's a link instead of a button:
https://zendesk.invisionapp.com/share/QZS1UPISHY5#/screens/357598308
that's why we use `a`

Reply:
`tabindex="0"` was inspired by this but not be necessary:
https://github.com/zendesk/copenhagen_theme/blob/master/templates/section_page.hbs#L32
I'll double check

Reply:
Should you change the language to match what's in the design mockup? So, instead of `see more`, it would say `Show more Help Centers`, or for below, `Show more Categories` and `Show more Sections`?

Reply:
@Jiyu-Zhang-Zendesk Since it doesn't cause navigation, it should be `button` semantically (for a11y). Then apply styling so it looks like a link.

Reply:
@VilhelmNielsen `tabindex="0"` is still needed here because our `a` doesn't have `href`. so without `tabindex` it won't be focusable

Reply:
@cmacaulay that's a good idea and I also mentioned it in the standup. not critical but might be nice to have.

Reply:
> @Jiyu-Zhang-Zendesk Since it doesn't cause navigation, it should be `button` semantically (for a11y). Then apply styling so it looks like a link.

I asked Augusto about this. This is a good point but we also have consideration over consistency since we already use hyperlinks for such cases in the section page:
https://github.com/zendesk/copenhagen_theme/pull/70
let's talk about this tomorrow. thanks